epuck_ros2_camera/src/pipuck_jpeg.c

```diff
@@ -0,0 +1,157 @@
+#include <assert.h>
```

Comment:
Did you write this file from scratch or does it come from somewhere?

Reply:
All GPU related files are written from scratch